### PR TITLE
fix(protocol): fix vault name and symbol validation bug with more unit tests

### PR DESCRIPTION
**Pull request by adaki2004**
- 1 bug (do not validate inputs for symbol and name)
- some additional tests
- fix commenting

### CODE REVIEW

File: packages/protocol/test/tokenvault/ERC20Vault.t.sol [hunk 1]
```diff
@@ -463,4 +520,162 @@ contract TestERC20Vault is TaikoTest {
...
+        // The following 2 scenarios would create a revert on the "destination" chain:
+        // A: when there is no name (also applies to symbol), we will use that exact same empty
+        // string decoded "" (LibBytes()).
+        // B: if the name() (also applies to symbol) low level call would fail, but we also use
+        // default "" (Base vault line 78).
+        //
+        // In these 2 scenarios, the LibBridgedToken.validateInputs(_srcToken, _srcChainId, _symbol,
+        // _name) -> Would fail.
+        // Either do not invalidate on empty string OR use other than "" as an error default retVal
+        // (in LibBytes) or default value.
```
Comment (adaki2004) [inline packages/protocol/test/tokenvault/ERC20Vault.t.sol:669]: @dantaik found a bug, please review the comment.

Comment (dantaik) [reply packages/protocol/test/tokenvault/ERC20Vault.t.sol:669]: I prefer remove the name and symbol validation logics.

**Comment by adaki2004** [comment]
@dantaik @davidtaikocha  Could one of you please check out this branch locally, and `forge fmt` ? 
Yesterday i was trying out something so i did a `foundryup` locally.. Since then the `forge fmt` command seemingly works differently than in our CICD.. so it seems it is making the forge format check fails here..
Or can we upgrade foundry with foundryup ? My current local version is:
`forge 0.2.0 (c486fca 2024-05-06T00:20:07.911403000Z)`

**Review by dantaik — APPROVED** [review]

**Comment by dantaik** [comment]
> @dantaik @davidtaikocha Could one of you please check out this branch locally, and `forge fmt` ? Yesterday i was trying out something so i did a `foundryup` locally.. Since then the `forge fmt` command seemingly works differently than in our CICD.. so it seems it is making the forge format check fails here.. Or can we upgrade foundry with foundryup ? My current local version is: `forge 0.2.0 (c486fca 2024-05-06T00:20:07.911403000Z)`

No worries about local `forge fmt`, now modified solidity files will be formatted automatically by a github workflow and all changes done by the formatter will be commited. 

Local formats are now temporary, you can safely ignore them.

**Review by davidtaikocha — APPROVED** [review]

**Pull request merged**